app/styles/themes/gothamist/_theme.mixins.scss

```diff
@@ -38,26 +33,25 @@
   display: inline-block;
   margin-right: var(--space-2);
   margin-left: calc(var(--space-1) * -1);
-  color: $c-gray;
+  color: rgb(var(--color-gray));
   font-size: $font-size;
   transform: translateY(.08em);
 }
 
 
 // Used for line wrapping text links.
 @mixin inline-accent(
-  $text-color: $c-body-color,
-  $underline-color: $c-body-color,
-  $hover-underline-color: $c-primary,
-  $hover-bg-color: $c-primary,
-  $hover-text-color: $c-white,
+  $text-color: rgb(var(--color-dark-gray)),
+  $underline-color: rgb(var(--color-dark-gray)),
+  $hover-underline-color: rgb(var(--color-reddish-orange)),
+  $hover-bg-color: rgb(var(--color-reddish-orange)),
+  $hover-text-color: rgb(var(--color-white)),
```

Comment:
Using these colors here is fine because we're in the gothamist theme so we know they exist, but we should still think about whether styles are used in the dark color set, because we still might want to use semantic tokens in those cases. (i don't know if that's the case here or not, just making a general comment here)